frontend/src/v5/store/containers/containers.redux.ts

```diff
@@ -16,18 +16,17 @@
  */
 
 import { createActions, createReducer } from 'reduxsauce';
-import { times } from 'lodash';
+import { cloneDeep, times } from 'lodash';
 import { Constants } from '@/v5/store/common/actions.helper';
 import { containerMockFactory } from './containers.fixtures';
-import { ContainerStatuses, IContainersState } from './containers.types';
-
-export interface IContainersActions {
-	setFilterQuery: (query: string) => any;
-}
+import { IContainersActionCreators, IContainersActions, IContainersState, ContainerStatuses } from './containers.types';
 
 export const { Types: ContainersTypes, Creators: ContainersActions } = createActions({
 	setFilterQuery: ['query'],
-}, { prefix: 'CONTAINERS/' }) as { Types: Constants<IContainersActions>; Creators: IContainersActions };
+	addFavourite: ['teamspace', 'projectId', 'containerId'],
+	removeFavourite: ['teamspace', 'projectId', 'containerId'],
+	toggleFavouriteSuccess: ['containerId'],
```

Comment:
I would change this to  `setFavouriteSuccess: ['containerId', 'isfavourite']` , otherwise is prone to errors. For example if you trigger twice in a row 'addFavourite' the firstime isFavourite is true and the second time isFavourite is false, even though you are expecting it to stay true.